test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestManagementFactory.java

```diff
@@ -1316,7 +1316,7 @@ public void testThreadMXBeanProxy() {
 				System.out.println("lockOwnerName ti1: " + ti1.getLockOwnerName() + " ti2: " + ti2.getLockOwnerName());
 				LockInfo li1 = ti1.getLockInfo();
 				LockInfo li2 = ti2.getLockInfo();
-				if (!li2.equals(li1)) {
+				if (li2 != li1) {
```

Comment:
It seems `LockInfo` inherits `equals()` from `Object`, so this seems ok, but perhaps a safer solution is to use `Objects.equals()`.

Reply:
Updated.